versions.csv

```diff
@@ -1,12 +1,12 @@
 AssemblyPath, Version
-iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.29.0-preview-004
-iothub\service\src\Microsoft.Azure.Devices.csproj, 1.27.0-preview-004
-shared\src\Microsoft.Azure.Devices.Shared.csproj, 1.16.0
-provisioning\device\src\Microsoft.Azure.Devices.Provisioning.Client.csproj, 1.4.0
-provisioning\transport\amqp\src\Microsoft.Azure.Devices.Provisioning.Transport.Amqp.csproj, 1.1.9
-provisioning\transport\http\src\Microsoft.Azure.Devices.Provisioning.Transport.Http.csproj, 1.1.6
-provisioning\transport\mqtt\src\Microsoft.Azure.Devices.Provisioning.Transport.Mqtt.csproj, 1.1.8
-security\tpm\src\Microsoft.Azure.Devices.Provisioning.Security.Tpm.csproj, 1.1.6
-provisioning\service\src\Microsoft.Azure.Devices.Provisioning.Service.csproj, 1.5.0
-digitaltwin\device\src\Azure.Iot.DigitalTwin.Device.csproj, 1.0.0-preview-001
-digitaltwin\service\src\Azure.Iot.DigitalTwin.Service.csproj, 1.0.0-preview-001
+iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.29.1-preview-001
+iothub\service\src\Microsoft.Azure.Devices.csproj, 1.27.1-preview-001
+shared\src\Microsoft.Azure.Devices.Shared.csproj, 1.20.0
```

Comment:
non "-preview" version updates are simply updates from "master". They will not be released again.